lua/entities/gmod_wire_expression2/core/http_cl.lua

```diff
@@ -0,0 +1,157 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
+
+]]
+
+--	Clientside HTTP is separate from serverside HTTP:
+--		Serverside HTTP allows greater speeds than clientside HTTP and it is
+--		up to the server owner to decide which extension should be enabled
+--		through a convar (found in core/http.lua)
+
+--	This library will mirror the functionality of core/http.lua as best as
+--	possible, making sure any existing e2s which make use of the http library
+--	are not broken by this extension
+
+-- should we use clientside HTTP requests?
+local cvar_useClient = CreateConVar("wire_expression2_http_client","1",FCVAR_ARCHIVE)
+
+E2Lib.clHTTP = {}
+local lib = E2Lib.clHTTP
+
+lib.netMsgID = "wire_expression2_cl_http" -- open to change
+lib.currentUID = 0
+
+-- number function(void)
+function lib:newUID()
```

Comment:
[`string WireLib.GetUUID()`](https://github.com/wiremod/wire/commit/fa46e272eb830ecdab615d83efec63a7b940f807#diff-8bd852ae59320acc0f3edd8d9f992c60R373)

We're *unlikely* to ever wrap around `self.currentUID`, but it's not guaranteed, and I'd rather the code used IDs which are never reused (because it makes bugs much harder to catch)